src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_importer.cljs

```diff
@@ -150,12 +161,16 @@
                       (swap! state assoc :error-message status-text))
         :mock-data (create-mock-methodconfs-import)}))})
 
-(defn render-import-overlay [workspace-id on-close on-import]
+(defn render-import-overlay [workspace-id on-close on-import selected-method]
```

Comment:
There's now the wrong number of arguments being passed in from method_configs_tab